heal/cli/heal_cli.py

```diff
@@ -0,0 +1,32 @@
+import logging
+
+import cdislogging
+import click
+
+import heal.cli.vlmd as vlmd
+
+
+@click.group()
+@click.option(
```

Comment:
this is good, maybe we should consider adding support of defining log level via option